### PR TITLE
Add components

### PR DESCRIPTION
**Pull request by yahyaelganyni1**
in this branch, I created a calculator using JSX and CSS with out the logic

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
+import './index.css';
 
 function App() {
```
Comment (vichuge) [inline src/App.js]: - [x] The App.js component should be class-based. I kindly invite you to work on it. 🐸

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Please, delete all unnecessary files like this. At this point is not necessary to have tests, but maybe in the future, you'll use them. 🐶 
image

- [ ] After installing the project and use `npm start` I have these issues, please work on it. This could be easily solved using `npx . eslint --fix`, please check the [linters config](https://github.com/microverseinc/linters-config/tree/master/react-redux) for more info
image

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (yahyaelganyni1) [reply src/App.js]: hello Victor I don't know what is the problem with the linters but they work fine on my PC I did a commit now can you tell me if this fixes  the linter problem

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @yahyaelganyni1  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**